common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -120,49 +159,199 @@ hfe_initializer_egypt_main = {
 							show_notification = no
 						}
 						create_leader = {
-							name = Amonhirkhopshef
+							name = "Amonhirkhopshef Mutatistep"
 							species = event_target:hfe_egypt_species
 							class = ruler
-							skill = 5
+							skill = 10
 							leader_age_min = 250
 							leader_age_max = 650
+							immortal = yes
+							gender = male
 							traits = {
 								trait = leader_trait_ruler_chosen
 								trait = hfe_ruler_trait_transcendent_0
 							}
 						}
 						assign_leader = last_created_leader
 					}
-					save_global_event_target_as = hfe_egypt_country
+					# Must initialize global designs here
+					add_global_ship_design = "NAME_Reaper" # Colossus!
```

Comment:
ai will never use it

Reply:
The AI can research and build a Colossus for a reason. They will use them, and even if they don't, a Colossus is really cool. Several of the expansions of the story that I wrote in involve planets cracking open. This is one of them.

Reply:
Ai cant use fallen empires ship designs, remove it.

Reply:
Yes they can. What do you think a FE builds when it awakes? It's a simple matter of adding it to the Country Type.

Reply:
FE can build it because have it in country definition.

Reply:
Human Fallen Empires have it in the country definition.

Reply:
Not then awaked

Reply:
That reminds me. Why don't the Kad Republic and Ju Ahets become Awakened Fallen Empires?

Reply:
Fallen empires have worse ai, also its paid feature, also new ship size mods compatibility and other reasons.